recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyreadline3-{{ version }}.tar.gz
-  sha256: ff3b5a1ac0010d0967869f723e687d42cabc7dccf33b14934c92aa5168d260b3
+  sha256: 6f3d1f7b8a31ba32b73917cefc1f28cc660562f39aea8646d30bd6eff21f7bae
 
 build:
   number: 0
```

Comment:
We should skip `py<35`, see https://github.com/pyreadline3/pyreadline3/commit/0549a7668c0c2a618993cee3646e5242b7b3bb6d